### PR TITLE
Feature/sleep class

### PR DESCRIPTION
**Pull request by st0nebreaker**
#### What's this PR do?
Adds tests, styled some long data on UI

#### Any background context you want to provide?
See annotations for why the Sleep testing wasn't working before

### CODE REVIEW

File: test/Sleep-test.js [hunk 1]
```diff
@@ -99,26 +103,38 @@ describe('Sleep', () => {
...
+	it('should be able to find the user who slept the most for a given date', () => {
+		let findSleepy = userSleep.findMostSleepUser(sleepData, '2019/07/21', userRepo);
 
-		expect(findBest).to.deep.equal([
-			1,  4,  6,  7,  8,  9, 10, 14,
-		 15, 17, 18, 20, 22, 23, 24, 28,
-		 29, 31, 33, 34, 35, 37, 38, 40,
-		 43, 44, 47
-	 ]);
+		expect(findSleepy).to.equal('Kristin Cruickshank slept more than any other user last night, 10.6 hours -- WOW!');
```
Comment (st0nebreaker) [inline test/Sleep-test.js:121]: Line 24 on this file, this user was missing from the smaller data being used for testing. So I added a 4th sample user, making sure it's the 23rd element in the array, the user matching the data being pulled in this method.

File: test/Sleep-test.js [hunk 2]
```diff
@@ -99,26 +103,38 @@ describe('Sleep', () => {
...
+	// it('should return the users with the best sleep week, quality over 3', () => {
+	// 	let findBest = userSleep.findBestSleepers(sleepData, '2019/07/21', userRepo);
+
+	// 	expect(findBest).to.deep.equal([
+	// 		1,  4,  6,  7,  8,  9, 10, 14,
+	// 	 15, 17, 18, 20, 22, 23, 24, 28,
+	// 	 29, 31, 33, 34, 35, 37, 38, 40,
+	// 	 43, 44, 47
+	//  ]);
+	// })
```
Comment (st0nebreaker) [inline test/Sleep-test.js:136]: This test won't work unless we add all those users in the long array (line 131-134). So either do that, or make the parameter on the test test a sample data, not the actual long data like we're trying in this testing file.

File: test/Sleep-test.js [hunk 3]
```diff
@@ -99,26 +103,38 @@ describe('Sleep', () => {
...
+
+	//TODO: add happy + sad test for .findBestSleepers method
+
```
Comment (st0nebreaker) [inline test/Sleep-test.js:139]: Left comments where happy or sad tests should be added!

Comment (carlymclift) [reply test/Sleep-test.js:139]: Awesome! thank you

**Pull request merged**